clusterloader2/api/types.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"time"
+)
+
+// Config is a structure that represents configuration
+// for a single test run
+type Config struct {
+	// Namespaces is a number of namespace used by the test.
+	Namespaces int32 `json: namespaces`
+	// Steps is a sequence of test steps.
+	Steps []Step `json: steps`
+	// TuningSets is a collection of tuning sets that can be used by steps.
+	TuningSets []TuningSet `json: tuningSets`
+}
+
+// Step is a structure that represents encapsulation
+// of some actions. This actions could be object declarations
+// or measurement usages.
```

Comment:
Please mention that exactly one of those should be non-empty.

Also mention that those will be running parallel.

Reply:
done